src/cmd-generate-release-meta

```diff
@@ -37,10 +41,12 @@ def get_extension(path, modifier, arch):
 parser = ArgumentParser()
 parser.add_argument("--workdir", help="cosa workdir")
 parser.add_argument("--build-id", help="build id")
+parser.add_argument("--distro", help="Distro selects stream defaults such as baseurl and format", choices=['fcos', 'rhcos'])
```

Comment:
```suggestion
parser.add_argument("--distro", help="Distro selects stream defaults such as baseurl and format", choices=['fcos', 'rhcos'], default='fcos')
```

Looks like it doesn't default to the first choice:

```
In [8]: parser = argparse.ArgumentParser()

In [9]: parser.add_argument('--foo', choices=['bar', 'baz'])
Out[9]: _StoreAction(option_strings=['--foo'], dest='foo', nargs=None, const=None, default=None, type=None, choices=['bar', 'baz'], help=None, metavar=None)

In [10]: parser.parse_args([])
Out[10]: Namespace(foo=None)
```

vs

```
In [14]: parser = argparse.ArgumentParser()

In [15]: parser.add_argument('--foo', choices=['bar', 'baz'], default='bar')
Out[15]: _StoreAction(option_strings=['--foo'], dest='foo', nargs=None, const=None, default='bar', type=None, choices=['bar', 'baz'], help=None, metavar=None)

In [16]: parser.parse_args([])
Out[16]: Namespace(foo='bar')
```

Reply:
Well...we could default, but it's kind of intentional not to.  We don't need to special case fcos here right now.

The idea here is that while some parts of cosa default to fcos, it's cleaner if it's more "mechanism" here.  So after this PR lands we could change the pipeline to pass `--distro fcos` explicitly?

Reply:
Ahh right OK. Just wanted to make sure this doesn't break FCOS as soon as it merges. But yeah, this doesn't.

Passing `--distro fcos` SGTM.

Reply:
One thing I wanted to comment that's related - while we could hardcode/default FCOS in cosa and have rhcos be overrides, I think as a general rule it's good to split mechanism from configuration/policy.

Because even if all we cared about was FCOS, we'd still have the problem of e.g. CI on pull requests may need to behave differently, as well as supporting multiple streams with different possible tweaks between them, etc.